server/logs/hyphaToken.js

```diff
@@ -0,0 +1,71 @@
+const web3 = require('../connections/web3')
+const { HyphaTokenABI, HyphaTokenAddress } = require('../abi/HyphaToken')
+const Network = require('../models/network')
+const User = require('../models/user')
+const Token = require('../models/token')
+const ProcessedTxs = require('../models/processedTxs')
+const netStatus = require('./network')
+
+module.exports = function () {
+  const HyphaTokenContract = new web3.eth.Contract(HyphaTokenABI, HyphaTokenAddress)
+
+  HyphaTokenContract.events.LogMint({fromBlock: netStatus.lastBlock}).on('data', async event => {
```

Comment:
Building some redundancy in here would be nice, like `fromBlock: netStatus.lastBlock-100` - at some point you may need to be sure your node isn't on a fork too, and lag a few blocks behind